### PR TITLE
Feat - added filter section for e com

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
*  Feat - added filter section for e com
* Watch the preview here:- https://deploy-preview-5--fast-cart.netlify.app/pages/filter.html
image

### CODE REVIEW

File: styles/filter.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.items{
+    padding: 1rem;
+}
```
Comment (haarsh24) [inline styles/filter.css:3]: define these property in util it as a class

**Review by haarsh24 — APPROVED** [review]

File: pages/filter.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+</head>
+
+<body>
+    <nav class="navbar-ecom">
+        <div class="brand-title-ecom">fastcart</div>
+        <a href="#" class="toggle-button-ecom">
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
```
Comment (xii-vi) [inline pages/filter.html:21]: what these spans are doing??

File: styles/filter.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+.filter{
+    justify-content: space-between;
+}
```
Comment (xii-vi) [inline styles/filter.css:7]: you could have made a genric class for using these

**Review by xii-vi — APPROVED** [review]
_Added a few comments. Rest is okay to merge._

**Pull request merged**